### DISCUSSION TITLE
how to use 'GraphicScene'?

### DISCUSSION THREAD

**Discussion by oreoreoreoreoreoreo** [3b1b/manim#1759] [Q&A] [answered]
I'm a biginner.
I tried to run a code including 'GraphicScene' class and an error 'NameError: name 'GraphScene' is not defined' occured
how should I do?

I would appreciate if you help.

**Comment by Rassska** [comment, marked as answer]
Hi! Great to see you!
Maybe, this one could help?
https://github.com/3b1b/manim/issues/1420

**Reply by TonyCrane** [reply]
And this: https://github.com/3b1b/manim/issues/1562#issuecomment-866980418
